src/pages/disliked/disliked.js

```diff
@@ -14,15 +18,32 @@ const Disliked = () => {
             <div className="video-card" key={i._id}>
               <div className="card-image">
                 <Link to={`/${i._id}`}>
-                  <img className="card-image" src={i.video_img} alt="" />
+                  <img
+                    className="card-image"
+                    src={i.video_img}
+                    onClick={
+                      isAuth ? () => addToHistory(i, dispatch) : undefined
+                    }
+                    alt={i.title}
+                  />
                 </Link>
               </div>
               <div className="card-title text">{i.title}</div>
               <div className="card-views">
                 <p>6k views</p>
                 <p className="hours-pading">| 4 hours ago</p>
               </div>
-              <button className="card-watch-button">Watch Later</button>
+              <button
+                className="card-watch-button"
+                onClick={() =>
+                  dispatch({
+                    type: "REMOVE_FROM_DISLIKES",
+                    payload: i,
+                  })
+                }
+              >
+                Remove from Disliked
+              </button>
             </div>
           ))}
       </div>
```

Comment:
here give some proper name so it make it readable , don't  understand what   mean for i

Reply:
Thanks for suggestion